src/containers/Profile/ProfileDetails.js

```diff
@@ -32,11 +74,40 @@ const ProfileDetailsComponent = props => {
     toggleNetworkSwitch,
     currentTab,
     setActiveTab,
-    setReviewsModalVisible
+    setReviewsModalVisible,
+    onEdit,
+    onCloseProgressBar,
```

Comment:
There should be an onClose prop on the banner component. This shouldn't live on the progress bar.